PluginCore/PluginCore/Helpers/DrawHelper.cs

```diff
@@ -11,6 +11,8 @@ public class DrawHelper
         /// </summary>
         public static Int32 MeasureDisplayStringWidth(Graphics graphics, String text, Font font)
         {
+            if (String.IsNullOrEmpty(text)) return 0;
```

Comment:
We don't need to run all the code below in this case, plus in the end it will throw an `IndexOutOfRangeException`.